src/lib/jsonParseV2/parseEntity.cpp

```diff
@@ -184,30 +177,29 @@ std::string parseEntity(ConnectionInfo* ciP, Entity* eP, bool eidInURL)
       if (r != "OK")
       {
         alarmMgr.badInput(clientIp, "parse error in context attribute");
-        eP->oe.fill(SccBadRequest, r, "BadRequest");
         ciP->httpStatusCode = SccBadRequest;
-
-        return eP->render(ciP->uriParamOptions, ciP->uriParam);
+        OrionError oe(SccBadRequest, r, "BadRequest");
+        return oe.toJson();
       }
     }
   }
 
   if (membersFound == 0)
   {
-    eP->oe.fill(SccBadRequest, "empty payload", "BadRequest");
+    alarmMgr.badInput(clientIp, "empty payload");
```

Comment:
I have had a look to this part of code but I don't find the log logic that overlaps with this...